app/views/campaign_pages/index.slim

```diff
@@ -11,7 +10,7 @@
           th Actions
           th
       tbody
-        - @campaign_pages.each do |page|
+        - [*@campaign_pages].each do |page|
```

Comment:
If you're protecting yourself from `nil`, you could also do `@campaign_pages].try(:each) do |page|`.  Personally, I'd make sure an array was assigned:

```ruby
  def index
    @campaign_pages = Search::PageSearcher.new(params).search || []
  end
```
... or even better, have `.search` always return an array.

```ruby
def search
 ... 
  @collection || []
end
```